packages/legacy/core/App/screens/Splash.tsx

```diff
@@ -214,7 +208,7 @@ const Splash: React.FC = () => {
 
         const newAgent = new Agent({
           config: {
-            label: store.preferences.walletName,
+            label: store.preferences.walletName || 'Aries Bifold',
```

Comment:
would it be better to set the wallet name if the preference is udnefined before initializing/loading the wallet?
```
if (!store.preferences.walletName) {
      dispatch({
        type: DispatchAction.UPDATE_WALLET_NAME,
        payload: [generateRandomWalletName()],
      })
  // retrieve updated `store`?
}
```

Reply:
Line 211 is just an additional backup in case somehow it's undefined after setting preferences from localstorage, but if it is undefined when preferences are being fetched it is being saved as `generateRandomWalletName()` in the that action. So in the flow of an older wallet running the new code it will be 'My Wallet - ####' not 'Aries Bifold'.